### ISSUE TITLE
improve template layouts

### ISSUE THREAD

**Issue by robertoostenveld** [fieldtrip/fieldtrip#2336]
Following https://github.com/fieldtrip/website/issues/723 I wanted to add the FieldLine alpha1 layout to the website, which made me realize that the https://www.fieldtriptoolbox.org/template/layout/ page was not well organized. The ToC on the right side was hard to use (since too many items) and did not distinguish between EEG, MEG and others. Also, some had the comment and scale at the bottom, some at the top, and some not at all.

With https://github.com/fieldtrip/website/commit/05fa045c40a4adf84a7bc48d34ca40690b8eff27 and some follow up commits I have refreshed the layouts and reorganized the documentation. In this process I realized that over time ft_prepare_layout defaults apparently have changed a bit and that some layouts (specifically the ones from lay files) have changed compared to the images there. 

It would be good to improve the quality of the template layouts. I discussed this with @schoffelen and we came up with the following:

1. Right now ft_prepare_layout  only includes a circle as head; we could also add realistic (egg-shaped) heads and outlines that go beyond the ears. We could also add double circles with a cross, as used in most of the original drawings of the easy cap layouts (with Oz on the inner and Iz on the outer circle).

2. We should recreate all lay files as mat files, so that they become independent of ft_prepare_layout changes. That means that the head outline gets hard-coded.

3. We should allow ft_prepare_layout to scale and shift the layout, when starting from an electrode set. Combined with a double-circle-with-cross outline, that should help get the electrodes projected at the desired 2D positions.

4. We should include realistic layouts for 10-20, 10-10 and 10-5 electrode arrangements, including a topographic interpolation that goes beyond the ears

5. We might want to change some of the circular layouts such that the topographic interpolation goes beyond the ears (i.e., in EEGLAB style).

**Comment by contsili** [comment]
Regarding suggestion 1. I believe that cfg.outline = 'doublecirclecross' should plot a head with double circles and a cross. Also, a realistic (egg-shaped) head would look like this?
![plot_sensors-in-MNE-Python](https://github.com/user-attachments/assets/34dc42ee-d7da-4410-bc99-2af946d7b758)

**Comment by robertoostenveld** [comment]
Yes. But the egg-shaped head by itself is not that much of an improvement on a circle; it becomes more relevant if you consider that some sensors are below the ears. 

This 
![image](https://github.com/user-attachments/assets/4dd0e722-18cc-40b6-a320-8844e6843db2)
uses that, but still as a sphere. 

This 
![image](https://github.com/user-attachments/assets/1c0fd50a-fe29-449c-8ba0-1b0eb9f88470)
uses it with a more realistic outline and mask
